manifests/tasks/move-model-to-root-dir.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: move-model-to-root-dir
+spec:
+  description: Moves model to the subdirectory, this means the location of the model is the same no matter the fetch method
+  params:
+  - name: model-name
+    type: string
+  - name: subdirectory
+    type: string
+  - name: src-model-relative-path
+    type: string
+  steps:
+  - name: copy-model-to-subdirectory
+    image: registry.access.redhat.com/ubi9/ubi
+    script: |
+      #!/usr/bin/env bash
+
+      set -Eeuo pipefail
+
+      mv $(workspaces.workspace.path)/$(params.subdirectory)/$(params.src-model-relative-path)/$(params.model-name) $(workspaces.workspace.path)/
+      rm -rf $(workspaces.workspace.path)/$(params.subdirectory)
+      mkdir $(workspaces.workspace.path)/$(params.subdirectory)
```

Comment:
@jackdelahunt you can try `mkdir -p ` and you'll no longer need to rm -rf. It basically throws no error if the directory already exists and then mv overwrites over the existing file.

Reply:
No because I am intentionally removing everything from the repo excluding the model, which is moved out before the rm